### ISSUE TITLE
The model's architecture

### ISSUE THREAD

**Issue by v-manhlt3** [Sreyan88/GAMA#22]
Hi,

First of all, thank you a lot for sharing the source code. I am keen on the model architecture, and I would like to modify the model's architecture and loss function for an academic research purpose. Do you have any plans to publish the model's architecture, especially the multi-layer aggregator and audio Q-former components?

Best regards

**Comment by Sreyan88** [comment]
Hi @v-manhlt3 ,

Thank You for your question. If I understand your query correctly, you need the model architecture to build on it. This codebase already has the same. Please see the code here: https://github.com/Sreyan88/GAMA/blob/main/hf-dev-train/transformers-main/src/transformers/models/llama/modeling_llama.py

If I am misunderstanding something, please let me know!

**Comment by v-manhlt3** [comment]
Thank you for getting back to me so quickly. I closed the issue as your response clarified my concern.

I appreciate your effort to publish source code and pre-trained models, It's a really cool project!
